README.md

```diff
@@ -8,6 +8,8 @@
 
 ## Making a plan
 
+![Wireframe](wireframe.png)
```

Comment:
Your wireframe image isn't showing up on github, the pathing is just a bit off. It should be the same path you would use to import into this file. 
![Wireframe](./assets/wireframe.png)